app/presenters/concerns/field_truncation.rb

```diff
@@ -5,6 +5,7 @@ module FieldTruncation
 
   def truncate_if_over_advertised_limit(field_name, field_value)
     limit = field_length(field_name)
+    return field_value if field_value.nil?
```

Comment:
can you add a test for this?

Reply:
Currently the `FieldTruncation` concern doesn't have its own test set - do you think this would be worth creating at this point, or should we test it through this api spec?

Reply:
On second thoughts, it's not a very testable module. I've added a test for a nil field in the `candidate_api_spec` to compensate